packages/react-instantsearch-hooks/scripts/transforms/rish-to-ris.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-shadow */
+
+export default function transform(file, api, options) {
+  const j = api.jscodeshift;
+  const printOptions = options.printOptions || { quote: 'single' };
+  const root = j(file.source);
+
+  const replaceImports = (from, to) => (root) =>
+    root
+      .find(j.ImportDeclaration)
+      .filter((path) => path.node.source.value === from)
```

Comment:
imports with specified paths need to be replaced too, probably doable with `startsWith` and a regex for the replace

Reply:
Ah yes I'll add tests for those too !